sources/packages/backend/apps/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -212,6 +235,47 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
     return found.length === locations.length;
   }
 
+  /**
+   * Check if location code is already registered for the institution.
+   * @param locationCode location code.
+   * @param locationId location id in case it is an update of the location.
+   * @param institutionId institution id in case it is a new location.
+   * @returns true in case there is already a location code for the institution.
+   */
+  async hasLocationCodeForInstitution(
+    locationCode: string,
+    locationId?: number,
+    institutionId?: number,
+  ): Promise<boolean> {
+    const institutionIdQueryBuilder = await this.repo
+      .createQueryBuilder("sublocation")
```

Comment:
Please use the name convention as `subQueryLocation`. It also avoids the code spell check to complain about it.

Reply:
ok

Reply:
Done.